bin/watch.js

```diff
@@ -34,6 +35,12 @@ require('yargs')
         describe: 'Webdav server url',
         type: 'string',
       })
+      .option('ssl', {
+        alias: 's',
+        describe: 'Suppress ssl issues',
+        type: 'boolean',
+        default: false
+      })
```

Comment:
let's name this option `strict-ssl-verification` and it's default should be `true`